### PR TITLE
Upload many

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Ready

- [x] upload many function uses upload/1 helper function
- [x] use folder structure on AWS: https://github.com/karmaradio/ev2/issues/56#issuecomment-351447092
- [x] fix startpack routes
- [x] fix delete startpack-file functionality

### CODE REVIEW

File: lib/ev2_web/controllers/startpack_controller.ex [hunk 1]
```diff
@@ -75,30 +75,34 @@ defmodule Ev2Web.StartpackController do
     uploaded_files = get_uploaded_files(startpack_map)
     delete_changeset =
       Accounts.change_startpack_delete(%Startpack{}, startpack_map)
+    offer = conn.assigns.offer
     image_changeset =
       Accounts.validate_startpack_uploads(%Startpack{}, startpack_params)
     case image_changeset.valid? do
       false ->
+        base_changesets =
+          determine_base_changesets(startpack_map, offer)
```
Comment (JMurphyWeb) [inline lib/ev2_web/controllers/startpack_controller.ex]: This was causing an error so I had to put it in here.

File: lib/ev2_web/templates/component/file_upload.html.eex [hunk 2]
```diff
@@ -6,11 +6,11 @@
   <p><%= assigns[:helper_text] %></p>
   <p>Accepted file types: <%= assigns[:file_types] %></p>
   <%= if assigns[:image_url] do %>
-    <a href=<%= assigns[:image_url] %>>
+    <!-- <a href=<%= assigns[:image_url] %>> -->
       <div class="relative k-min-h-10">
         <img src=<%= get_thumbnail(assigns[:image_url]) %> class="<%= get_thumbnail_style(assigns[:image_url]) %>" />
       </div>
-    </a>
+    <!-- </a> -->
```
Comment (JMurphyWeb) [inline lib/ev2_web/templates/component/file_upload.html.eex]: I made these small icons not downloadable. Users can view their downloads at the top of the startpack.

File: lib/ev2_web/views/view_helpers.ex [hunk 3]
```diff
@@ -71,15 +71,15 @@ defmodule Ev2Web.ViewHelpers do
...
     end
   end
 
   def get_thumbnail_style(url) do
     case String.ends_with?(url, ".pdf") do
-      true -> "absolute k-left--05 k-h10 mb3 db k-w3 top--1"
-      false -> "h5 mb3 db"
+      _true -> "absolute k-left--05 k-h10 mb3 db k-w3 top--1"
+      # false -> "h5 mb3 db"
     end
```
Comment (JMurphyWeb) [inline lib/ev2_web/views/view_helpers.ex]: For startpack view these were not loading anything for .png and .jpg, so I have just forced it to be the 'file' icon.

The issue could be due to new permissions on our aws files?

File: lib/ev2_web/templates/component/file_upload.html.eex (hunk 2 above)
Comment (jackcarlisle) [reply lib/ev2_web/templates/component/file_upload.html.eex]: Cool, can we remove the comments then?

File: lib/ev2_web/views/view_helpers.ex [hunk 4]
```diff
@@ -71,15 +71,15 @@ defmodule Ev2Web.ViewHelpers do
 
   def get_thumbnail(url) do
     case String.ends_with?(url, ".pdf") do
-      true -> "/images/static/file.png"
-      false -> url
+      _ -> "/images/file.png"
+      # false -> url
```
Comment (jackcarlisle) [inline lib/ev2_web/views/view_helpers.ex]: can this be removed?

File: lib/ev2_web/views/view_helpers.ex [hunk 5]
```diff
@@ -71,15 +71,15 @@ defmodule Ev2Web.ViewHelpers do
...
     end
   end
 
   def get_thumbnail_style(url) do
     case String.ends_with?(url, ".pdf") do
-      true -> "absolute k-left--05 k-h10 mb3 db k-w3 top--1"
-      false -> "h5 mb3 db"
+      _true -> "absolute k-left--05 k-h10 mb3 db k-w3 top--1"
+      # false -> "h5 mb3 db"
```
Comment (jackcarlisle) [inline lib/ev2_web/views/view_helpers.ex]: same for this?

File: lib/ev2_web/views/view_helpers.ex (hunk 3 above)
Comment (jackcarlisle) [reply lib/ev2_web/views/view_helpers.ex]: Possibly, need to take a closer look

**Pull request merged**